07-systems_access_policy.md

```diff
@@ -41,7 +41,7 @@ Access to Health Note systems and applications is limited for all users, includi
 ## 7.2 Access Establishment and Modification
 
 1. Requests for access to Health Note systems and applications are made formally using the following process:
-  1. A Health Note workforce member initiates the access request by filling out a **Systems Access Form** [here](https://docs.google.com/forms/d/e/1FAIpQLSdus2wN2WHjbj0JBal2aG6Z8NmguTP7am1GCmZAgPvb7zSKaw/viewform?usp=pp_url&entry.768761467=Pending).
+  1. A Health Note workforce member initiates the access request by filling out a [**Systems Access Form**](https://docs.google.com/forms/d/e/1FAIpQLSdus2wN2WHjbj0JBal2aG6Z8NmguTP7am1GCmZAgPvb7zSKaw/viewform?usp=pp_url&entry.768761467=Pending).
      * User identities must be verified prior to granting access to new accounts.
      * Identity verification must be done in person where possible; for remote employees, identities must be verified over the phone.
```

Comment:
We are a remote company so I am wondering if we should just update to just have that as well as update copy for "verified over the phone" since we have not been doing that lately

Reply:
Yes, let's update it to something like
* As a remote-only company, identifies must be verified over the phone or video conference.